### PR TITLE
Updated openshift templates

### PR DESCRIPTION
**Pull request by quarckster**

### CODE REVIEW

File: openshift/templates/compliance-backend-bc.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: builder-insights-compliance-backend
+
+parameters:
+- name: NAMESPACE
```
Comment (dLobatog) [inline openshift/templates/compliance-backend-bc.yaml]: 2 keys in the same level, no indentation, is this correct?

Comment (dLobatog) [reply openshift/templates/compliance-backend-bc.yaml]: There are several instances of this in this 'parameters'  key

File: openshift/templates/compliance-backend.yml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          - name: POSTGRESQL_SHARED_BUFFERS
+            value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: RAILS_ENV
+            value: "${RAILS_ENV}"
+          resources:
+            limits:
+              memory: "${MEMORY_LIMIT}"
+
+parameters:
+- name: NAMESPACE
```
Comment (dLobatog) [inline openshift/templates/compliance-backend.yml]: ditto

File: openshift/templates/compliance-consumer.yaml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          - name: RAILS_ENV
+            value: "${RAILS_ENV}"
+          - name: KAFKAMQ
+            value: "${KAFKAMQ}"
+          resources:
+            limits:
+              memory: "${MEMORY_LIMIT}"
+
+parameters:
+- name: NAMESPACE
```
Comment (dLobatog) [inline openshift/templates/compliance-consumer.yaml]: ditto

File: openshift/templates/compliance-db.yaml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+          - name: POSTGRESQL_MAX_CONNECTIONS
+            value: "${POSTGRESQL_MAX_CONNECTIONS}"
+          - name: POSTGRESQL_SHARED_BUFFERS
+            value: "${POSTGRESQL_SHARED_BUFFERS}"
+          resources:
+            limits:
+              memory: "${MEMORY_POSTGRESQL_LIMIT}"
+
+parameters:
+- name: NAMESPACE
```
Comment (dLobatog) [inline openshift/templates/compliance-db.yaml]: ditto

File: openshift/templates/compliance-backend-bc.yaml (hunk 1 above)
Comment (quarckster) [reply openshift/templates/compliance-backend-bc.yaml]: it's required by `ocdeployer` https://github.com/RedHatInsights/e2e-deploy#how-do-template-variables-work.:
> By default, the 'NAMESPACE' variable is passed to all templates, this corresponds to the project name.
It just needs to be defined in parameters.

Comment (quarckster) [reply openshift/templates/compliance-backend-bc.yaml]: The indentation is correct. The template is deployed successfully.

**Pull request merged**